### PR TITLE
Feature/4575 restore tool

### PR DESCRIPTION
**Pull request by acn-sbuad**
Data restore implemented. 
CLI application project set up.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -348,3 +348,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# Custom
+/AltinnReStorage/appsettings.json
```
Comment (altinnadmin) [inline .gitignore:353]: Why is appsettings.json added to .gitignore?

File: AltinnReStorage/AltinnReStorage.csproj [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  <ItemGroup>
+    <Compile Remove="Attributes\StephAttribute.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="2.2.2" />
+    <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.ResourceManager.Storage" Version="1.0.0-preview.1" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.4.4" />
```
Comment (altinnadmin) [inline AltinnReStorage/AltinnReStorage.csproj:16]: 12.5.0 is latest. Seems to contain some important fixes.
https://github.com/Azure/azure-sdk-for-net/blob/Azure.Storage.Blobs_12.5.0/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

File: .gitignore (hunk 1 above)
Comment (acn-sbuad) [reply .gitignore:353]: It contains sensitive data such as subscription ids for all of our subscriptions.
We discussed making this file available on AltinnPedia or somewhere else.. Using mockdata and checking the file into the repo doesn't make much sense either

File: AltinnReStorage/AltinnReStorage.csproj (hunk 2 above)
Comment (acn-sbuad) [reply AltinnReStorage/AltinnReStorage.csproj:16]: :+1:

**Pull request merged**